src/types/UploaderType.ts

```diff
@@ -0,0 +1,25 @@
+import YoutubeIcon from '@src/assets/icons/youtubeIcon';
+import InstagramIcon from '@src/assets/icons/InstagramIcon';
+import BlogIcon from '@src/assets/icons/BlogIcon';
+
+export enum UploaderType {
+    Youtuber = 'YOUTUBER',
+    Blogger = 'BLOGGER',
+    Instagramer = 'INSTAGRAMER',
+    Editor = 'EDITOR',
+    None = 'NONE'
+}
+
+const {
+    Youtuber,
+    Blogger,
+    Instagramer,
+    Editor,
+    None
+} = UploaderType;
+
+export const UploaderTypeIcon = {
+    [None]: YoutubeIcon,    // 테스트 후 Youtuber로 변경해야 함
+    [Instagramer]: InstagramIcon,
+    [Blogger]: BlogIcon,
+};
```

Comment:
SelectsRequest.ts 파일보고 관련된 소스인것같아서 여기로 위치를 옮겨서
card에서 사용했어요

Reply:
아하 이 부분은
lib로 빼는게 좋을 것 같아요!
해당 기능이 type보단 함수처럼 작동하기 때문입니다.
이 함수만 lib/user 경로로 빼서 작성해주시면 됩니당

Reply:
넵 수정했습니다. 이부분은 맞게 작성한건가요??

Reply:
음.. 아래처럼 바꿔서 사용하는게 좋을 것 같아요!
```suggestion
export const getUploaderTypeIconComponent = (uploaderType: UploaderType) => (
  {
    [None]: <YoutubeIcon />,    // 테스트 후 Youtuber로 변경해야 함
    [Instagramer]: <InstagramIcon />,
    [Blogger]: <BlogIcon />,
  }[uploaderType] || () => <></>
);
```